element_deeplabcut/model.py

```diff
@@ -529,14 +343,11 @@ def make(self, key):
                           test_error_p=results['Test error with p-cutoff']))
 
 
-# ---- Model Inference pipeline ----
-
-
 @schema
-class PoseEstimationTask(dj.Manual):
+class EstimationTask(dj.Manual):

```

Comment:
```suggestion
class PoseEstimationTask(dj.Manual):

```
I'd still prefer the name `PoseEstimationTask` for clarity

Reply:
Not sure why you were seeing an outdated version here after [this commit yesterday](https://github.com/CBroz1/element-deeplabcut/pull/5/commits/b84254785e42d2ae07e344bbf11bb84e2979ee51) reverted to `PoseEstimation`